.github/workflows/mkdocs.yml

```
@@ -6,9 +6,17 @@ on:
 
+permissions:
+  contents: write
+  pages: write
+  id-token: write
+
 jobs:
   deploy:
     runs-on: ubuntu-latest
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
```

actionlint (1.7.8)

property "deployment" is not defined in object type {}

(expression)